### DISCUSSION TITLE
Removing metadata

### DISCUSSION THREAD

**Discussion by coastalnomad** [Homebrew/discussions#5163] [Everyday usage]
### Output of `brew config`

```shell
I have no idea how to use this discussion but i want to remove metadata from photos and video. Does Homebrew work for this?
```

### Output of `brew doctor`

```shell
No idea what this means
```

### Description of issue

Again, no idea what this is

**Comment by p-linnane** [comment]
We distribute software. You'll need to find software that does what you want, and then see if we package it. We are not a discoverability service.

**Reply by coastalnomad** [reply]
I fell down a Homebrew rabbit hole and ended up here. I'm having a hard time finding software that removes exif data

**Reply by gromgit** [reply]
No idea what a "Homebrew rabbit hole" is, but since you seem to not have installed Homebrew yet, you _could_ search the list of [all Homebrew formulae](https://formulae.brew.sh/formula/) for "exif" and see if anything piques your interest. If you're running a macOS box, you might additionally want to search the list of [all Homebrew casks](https://formulae.brew.sh/cask/) as well.

But the above lists have necessarily brief descriptions, so you might be glossing over something that _does_ meet your requirements, but doesn't have "exif" in the description. That's @p-linnane's point: Google / DuckDuckGo / your favorite public search engine is a _far_ better way to find the name of what you need, with as detailed a description as you want, _then_ see if someone's packaged it for general consumption on Homebrew.

Also, to install _anything_ you see in the above lists, you have to [first install Homebrew-the-package-manager](https://brew.sh/), then use _that_ to install the software you want.

**Reply by coastalnomad** [reply]
I'm not a programmer so a lot just goes over my head, sorry lol. This is where i started, just trying to understand how it works, and if it's safe: https://sunknudsen.com/privacy-guides/how-to-remove-exif-metadata-from-images-on-macos
Thanks for your patience!

**Reply by p-linnane** [reply]
This makes it easier to help you now that we understand you are trying to install the `exiftool` package. The commands provided in that article need to be ran in your terminal. So open Terminal.app, paste in `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`. Follow the prompts, including the parts after the install completes. Once that's done you can run `brew install exiftool` and you should be good to go. The article you linked has a video tutorial which should also be helpful.

**Reply by coastalnomad** [reply]
ok thank you!
